.gitignore

```
@@ -35,6 +35,6 @@ convoy.json
 docker-compose.yml
 compose.yml
 typesense_data
-postgres_data
+postgres_data**
```

Won't the below work instead?
```suggestion
postgres_data/
```